subsys/usb/usb_descriptor.c

```diff
@@ -622,6 +623,18 @@ static struct dev_common_descriptor common_desc = {
 				CONFIG_HID_INTERRUPT_EP_MPS),
 			.bInterval = 0x09,
 		},
+
+		.if1_int_ep = {
+			.bLength = sizeof(struct usb_ep_descriptor),
+			.bDescriptorType = USB_ENDPOINT_DESC,
+			.bEndpointAddress = CONFIG_HID_OUT_EP_ADDR,
+			.bmAttributes = USB_DC_EP_INTERRUPT,
+			.wMaxPacketSize =
+				sys_cpu_to_le16(
+			CONFIG_HID_INTERRUPT_EP_MPS),
```

Comment:
In the web it looks like not aligned like previous structure

Reply:
I am not sure why it is happening